pkg/backend/httpstate/backend.go

```diff
@@ -1080,13 +1090,22 @@ func (b *cloudBackend) apply(
 	}
 
 	// Create an update object to persist results.
-	update, version, token, err := b.createAndStartUpdate(ctx, kind, stack, &op, opts.DryRun)
+	update, updateMeta, err := b.createAndStartUpdate(ctx, kind, stack, &op, opts.DryRun)
 	if err != nil {
 		return nil, nil, result.FromError(err)
 	}
+	updateMeta.messages = append(updateMeta.messages, apitype.Message{
+		Message: "Good morning, user! How are you today?",
+	})
+	displayMessages := func() {
+		for _, msg := range updateMeta.messages {
+			fmt.Printf("[service] %s\n", msg.Message)
```

Comment:
This probably needs to be updated based on the new branding:

```suggestion
			fmt.Printf("[Pulumi Cloud] %s\n", msg.Message)
```

Would we still display `[Pulumi Cloud]` if self-hosted?

Should we drop the prefix altogether?

Reply:
I recommend adding a prefix of some sort to convey that it's from the backend rather than the hardcoded into the CLI.

thoughts @susanev